cmds/houndd/main.go

```diff
@@ -13,6 +14,7 @@ import (
 	"strings"
 	"syscall"
 
+	"github.com/blang/semver"
```

Comment:
we've just added support for gomodules — are you ok with adding this to our `go.mod` file? I believe you will need to rebase master into this branch and force-push to see it.